.github/workflows/check-pr-jvm-godot-library.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check PR - JVM Godot Library
+on:
+  pull_request:
+    paths:
+      - 'kt/buildSrc/**'
+      - 'kt/gradle/**'
+      - 'kt/godot-library/**'
+      - 'kt/godot-runtime/**'
+      - '.github/workflows/check-pr-jvm-godot-library.yaml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Godot JVM module.
+        uses: actions/checkout@v2
+      - name: Checkout submodules
+        uses: snickerbockers/submodules-init@v4
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
same here, make it a matrix property so we can easily add new JVM targets to test against.